### ISSUE TITLE
Menu overlay: Keyboard navigation - some keys are not supported yet

### ISSUE THREAD

**Issue by RohdeSchwarz-SDC** [primefaces/primefaces#12456]
### Description

The `<p:menuButton>` supports many keys for navigation, being a well-designed component.

On the other hand a `<p:commandButton>` with `<p:menu>` mimics the same behaviour as `<p:menuButton>` (see PrimeFaces Showcase > Menu > Overlay).

However some keys are not supported yet:
* `[POS1] aka [HOME]` or `[PageUp]` - jump to the first menu item
* `[END]` or `[PageDown]` - jump to the last menu item
* `[ArrowUp]` - when focuse is on first item -> go to last item

### Describe the solution you would like

For consistency reasons the keys listed above should work on a menu overlay in the same way as on a menuButton.

**Comment by melloware** [comment]
I will take a look at this one. IT should be consistent if it can be.

**Comment by melloware** [comment]
Thanks for the report it is consistent now with WCAG standards.
